### PR TITLE
Use new translation files

### PR DESCRIPTION
**Pull request by jdhoffa**
depends on 2DegreesInvesting/create_interactive_report/pull/70

### CODE REVIEW

**Comment by jdhoffa** [comment]
I notice CI test are failing due to missing GH PAT?

**Comment by cjyetman** [comment]
@jdhoffa yes, @maurolepore hasn't yet got the secrets setup properly so that forks of this repo can successfully trigger the Actions

**Comment by jacobvjk** [comment]
> I notice CI test are failing due to missing GH PAT?

which is why we are currently pushing upstream rather than to our forks in this repo... @jdhoffa

**Comment by jdhoffa** [comment]
@jacobvjk copy that, i will do so in the future

File: web_tool_script_3.R [hunk 1]
```diff
@@ -230,16 +230,25 @@ if (file.exists(file.path(results_path, portfolio_name, "Stress_test_results_IPR
 }
 
 
-indicies_equity_results_portfolio <- read_rds(file.path(data_location_ext, "Indices_equity_portfolio.rda"))
-indicies_bonds_results_portfolio <- read_rds(file.path(data_location_ext, "Indices_bonds_portfolio.rda"))
+indices_equity_results_portfolio <- read_rds(file.path(data_location_ext, "Indices_equity_portfolio.rda"))
```
Comment (jacobvjk) [inline web_tool_script_3.R:233]: this is eye candy

**Review by jacobvjk — APPROVED** [review]
LGTM

**Pull request merged**